res/Sectors/M1105/blaskon.txt

```diff
@@ -0,0 +1,474 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:12:11+00:00
+
+# Blaskon
+# 6,2
+
+# Name: Blaskon (Hv)
+
+# Milieu: M1105
+
+# Credits: Blaskon sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Taxcolask
+# Subsector B: Aawrul 
+# Subsector C: Lotrutan
+# Subsector D: 
+# Subsector E: Valaar
+# Subsector F: Inasko
+# Subsector G: Ldoocox
+# Subsector H: Anix Nuno
+# Subsector I: Orkanax
+# Subsector J: Laxsetker
+# Subsector K: Guuqal
+# Subsector L: 
+# Subsector M: 
+# Subsector N: Kipnasko
+# Subsector O: 
+# Subsector P: Dradrip
+
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"